Index: Backend/app.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>from flask import Flask\r\n\r\napp = Flask(__name__)\r\n\r\n\r\n@app.route(\"/\")\r\ndef home_page():\r\n    return {\"Home Page\": [\"Profile\", \"Predict Heart Diseases\", \"Doctors List\", \"Upload ECG\"]}, 200\r\n\r\n\r\n@app.route(\"/profile\")\r\ndef profile_page():\r\n    return {\"Profile Page\": [\"Name\", \"Username\", \"Email\", \"Gender\", \"DOB\"]}, 200\r\n\r\n\r\n@app.route(\"/predict-heart-disease\")\r\ndef medical_info_page():\r\n    return {\"Predict Heart Disease Page\": [\"age\", \"gender\", \"chest pain type\", \"...\"]}, 200\r\n\r\n\r\n@app.route(\"/doctors\")\r\ndef doctors_page():\r\n    return {\"Doctors List\": [\"Doctor ID\", \"Doctor Name\", \"Hospital\", \"Contact\"]}, 200\r\n\r\n\r\n@app.route(\"/ECG-Page\")\r\ndef ecg_page():\r\n    return {\"ECG Upload\": [\"Browse\"]}, \r\n  \r\n@app.route(\"/About\")\r\ndef about_page():\r\n    return \"About\"\r\n\r\n\r\nif __name__ == '__main__':\r\n    app.run()\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Backend/app.py b/Backend/app.py
--- a/Backend/app.py	(revision 788a27c7b61d6ca035e0593c1a25716834e66062)
+++ b/Backend/app.py	(date 1646586013713)
@@ -25,7 +25,7 @@
 
 @app.route("/ECG-Page")
 def ecg_page():
-    return {"ECG Upload": ["Browse"]}, 
+    return {"ECG Upload": ["Browse"]}, 200
   
 @app.route("/About")
 def about_page():
@@ -33,4 +33,4 @@
 
 
 if __name__ == '__main__':
-    app.run()
+    app.run(debug=True)
\ No newline at end of file
Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/main.py
new file mode 100644
--- /dev/null	(date 1646586013729)
+++ b/main.py	(date 1646586013729)
@@ -0,0 +1,16 @@
+# This is a sample Python script.
+
+# Press Shift+F10 to execute it or replace it with your code.
+# Press Double Shift to search everywhere for classes, files, tool windows, actions, and settings.
+
+
+def print_hi(name):
+    # Use a breakpoint in the code line below to debug your script.
+    print(f'Hi, {name}')  # Press Ctrl+F8 to toggle the breakpoint.
+
+
+# Press the green button in the gutter to run the script.
+if __name__ == '__main__':
+    print_hi('PyCharm')
+
+# See PyCharm help at https://www.jetbrains.com/help/pycharm/
